app/components/Layout/Header.js

```diff
@@ -15,62 +16,52 @@ const Header = props => (
           </Link>
           <h1>CleanBC Industrial Incentive Program</h1>
         </div>
-        {!props.isLoggedIn && (
-          <div className="login-btns header-right">
-            <Row>
-              <Col>
-                <Link
-                  href={{
-                    pathname: '/dummy-login'
-                  }}
-                >
-                  <Button style={{marginRight: '10px'}} variant="outline-light">
-                    Register
-                  </Button>
-                </Link>
-                <Link
-                  href={{
-                    pathname: '/dummy-login'
-                  }}
-                >
-                  <Button variant="outline-light">Login</Button>
-                </Link>
-              </Col>
-            </Row>
-          </div>
-        )}
+        <div className="login-btns header-right">
+          <Row>
+            <Col>
+              {props.session ? (
+                <Form.Row>
+                  <Col>
+                    <Form action="/user-profile" method="get">
+                      <Button type="submit" variant="outline-light">
+                        Profile
+                      </Button>
+                    </Form>
+                  </Col>
+                  <Col>
+                    <Form action="/logout" method="post">
+                      <Button type="submit" variant="outline-light">
+                        Logout
+                      </Button>
+                    </Form>
```

Comment:
@matthieu-foucault this triggers a full re-render and the Location header is respected by the browser as a result